operator/v1/types_network.go

```diff
@@ -429,6 +430,15 @@ type HybridOverlayConfig struct {
 }
 
 type IPsecConfig struct {
+	// Mode defines the ipsec configuration level, 'Off' disable ipsec on the node level,
```

Comment:
Comment needs to be updated based on change from `Off` to `Disabled`.

Also, comment should start with json tag name, not Go field name